lms/envs/common.py

```diff
@@ -2701,3 +2701,10 @@
     'options': {},
 }
 PROCTORING_SETTINGS = {}
+
+#### Custom Courses for EDX (CCX) configuration
+
+# This is an arbitrary hard limit.
+# The reason we introcuced this number is because we do not want the CCX
+# to compete with the MOOC.
+CCX_MAX_STUDENTS_ALLOWED = 200
```

Comment:
Configuring the number of items to show on a page is a UX decision and it's a bit insane to make all those things site wide configurable.

Limiting the size of a CCX potentially makes more sense as a sitewide configuration option. I'm curious what product or marketing decisions have led to a limit of 200. If it's truly arbitrary, I think it merits a bit more discussion. We shouldn't be adding features without having good justifications for the decisions we're making.